docs/ref-info/glossary-uid.md

```diff
@@ -112,6 +113,9 @@ import Link from '@docusaurus/Link';
 <dt><MdxJumpAnchor id="gl-api-secret">API secret</MdxJumpAnchor></dt>
 <dd>See <a href="#gl-client-secret">client secret</a>.</dd>
 
+<dt><MdxJumpAnchor id="gl-app-name">App name</MdxJumpAnchor></dt>
+<dd>The Android application ID,  the iOS app store ID, or the iOS bundle identifier.</dd>
```

Comment:
is that extra spaces before "the iOS app store ID" ?

Reply:
thanks, fixed